pegasus/sites.v3/code.org/public/css/tutorial-promo.css

```diff
@@ -24,15 +24,22 @@
   padding: 5px;
   top: 70px;
 }
+/* left-tutorial and right-tutorial are both usually aligned to the left side, different alignments are needed for dance and starwars */
 
-.left-tutorial {
+.left-tutorial  {
   right: 20px;
 }
-
+.left-tutorial.inverted{
```

Comment:
*nit* - Maybe call this `right-to-left` instead instead of `inverted` since that is the context for this class.

Reply:
Yeah that makes sense, I changed it